### PR TITLE
chore: add function tests

### PR DESCRIPTION
**Pull request by Rachelint**
# Which issue does this PR close?

Closes #

# Rationale for this change
Add testcases for existing udfs in ceresdb.

# What changes are included in this PR?
Add testcases in integrated test for udfs(thetasketch_distinct and time_bucket).

# Are there any user-facing changes?
None.

# How does this change test
They are tests .

### CODE REVIEW

File: tests/cases/02_function/time_bucket.sql [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+SELECT time_bucket(`timestamp`, 'P1W') FROM `02_function_time_bucket_table`;
+SELECT time_bucket(`timestamp`, 'P1D') FROM `02_function_time_bucket_table`;
+SELECT time_bucket(`timestamp`, 'PT1H') FROM `02_function_time_bucket_table`;
+SELECT time_bucket(`timestamp`, 'PT1M') FROM `02_function_time_bucket_table`;
+SELECT time_bucket(`timestamp`, 'PT1S') FROM `02_function_time_bucket_table`;
+
+-- Test various parameters.
+SELECT time_bucket(`timestamp`, 'PT1H', 'yyyy-MM-dd HH:mm:ss') FROM `02_function_time_bucket_table`;
+SELECT time_bucket(`timestamp`, 'PT1H', 'yyyy-MM-dd HH:mm:ss', '+0800') FROM `02_function_time_bucket_table`;
+SELECT time_bucket(`timestamp`, 'PT1H', 'yyyy-MM-dd HH:mm:ss', '+0800', 'yyyy-MM-dd HH') FROM `02_function_time_bucket_table`;
```
Comment (waynexia) [inline tests/cases/02_function/time_bucket.sql:33]: We do not support customizing format, maybe add a comment about this

**Review by waynexia — APPROVED** [review]

**Comment by waynexia** [comment]
Good job!

I recognize we lack some documents about those UDFs when reviewing it. I'm unfamiliar with `time_bucket` and need to read the source to figure out what those parameters mean.

**Comment by Rachelint** [comment]
> Good job!
> 
> I recognize we lack some documents about those UDFs when reviewing it. I'm unfamiliar with `time_bucket` and need to read the source to figure out what those parameters mean.

Yes, docs about udfs are necessary.

File: tests/cases/02_function/time_bucket.sql (hunk 1 above)
Comment (Rachelint) [reply tests/cases/02_function/time_bucket.sql:34]: > We do not support customizing format, maybe add a comment about this

Done.

**Pull request merged**